### PR TITLE
Add suggesters to bioentities

### PR DESCRIPTION
**Pull request by alfonsomunozpomer**
This started as a “let’s just add a script to create the suggesters” and be over with it, but while doing that I noticed a few shortcomings here and there and decided to fix several issues along the way.

I added `g++` to the test module container which is necessary to install Pandas, since it’s got some Cython parts. I also added PyYAML and `property_weights.yaml` to the test container, otherwise documents weren’t being added at all. I think it would be good to add an explicit test to check for Solr docs. Now this is implicitly covered by the suggester tests returning one or more results when querying for known terms.

I fixed some display messages and a few typos I found.

Other than the above, I added a couple of scripts to create the suggesters we use in (bulk) Expression Atlas and Single Cell Expression Atlas. I added them as different scripts so we haven’t got extra cruft in either service. I added tests that check for the `/suggest` request handler, that they return good data or no suggestions if the searched term doesn’t match anything in the collection.

### CODE REVIEW

**Comment by pcm32** [comment]
Useful additions! 

Could you please:
1.- rebase on develop or resolve the conflicts as you see fit
2.- move stuff concerning SCXA to https://github.com/ebi-gene-expression-group/index-scxa/

Thanks!

File: run_tests_in_containers.sh [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+docker network create mynet
+docker run --rm --net mynet --name my_solr -v $(pwd)/lib/solr-ontology-update-processor-1.1.jar:/opt/solr/server/solr/lib/solr-ontology-update-processor-1.1.jar -d -p 8983:8983 -t solr:7.1-alpine -DzkRun -Denable.runtime.lib=true -m 2g
 docker build -t test/index-gxa-module .
 sleep 20
 
 docker exec -it --user=solr my_solr bin/solr create_collection -c gxa-analytics-v1
-docker run -i --net mynet -v $( pwd )/tests:/usr/local/tests -e SOLR_HOST=$SOLR_HOST --entrypoint=/usr/local/tests/run_tests_inside_container.sh test/index-gxa-module
+docker run --rm -i --net mynet -v $( pwd )/tests:/usr/local/tests -e SOLR_HOST=$SOLR_HOST --entrypoint=/usr/local/tests/run_tests_inside_container.sh test/index-gxa-module
+
+docker stop my_solr
```
Comment (pcm32) [inline run_tests_in_containers.sh:13]: I would leave this as optional, some of us use this to simply lift the setup and have some dummy data to test queries or loading scripts when making changes to the scripts here.

**Comment by alfonsomunozpomer** [comment]
> 2.- move stuff concerning SCXA to https://github.com/ebi-gene-expression-group/index-scxa/

Then shall we replicate the scripts to create the configset, collection and schema in that repo? It seems to me that it would be somewhat brittle... Maybe we should create an `index-bioentities` repo and include it in both `index-gxa` and `index-scxa` as a Git submodule?

**Comment by pcm32** [comment]
Yeah, sounds good to have bioentities as a separate repo for loading.

**Comment by alfonsomunozpomer** [comment]
> Yeah, sounds good to have bioentities as a separate repo for loading.

Done. Have a look at the other repo and let me know how you want to handle this. Either using it in both `index-scxa` as a submodule or as a separate thing altogether. I’d favour the latter but I don’t know if you had other plans.

**Comment by pcm32** [comment]
Closing in favour of #13 which resolves the many conflicts more cleanly.

**Pull request closed without merging**